infra/modules/gcp-oauth-long-access-strategy-read/main.tf

```diff
@@ -0,0 +1,10 @@
+locals {
+  secret_id = "PSOXY_ACCESS_TOKEN_${var.function_name}"
+}
+
+# secret to hold a long-lived access token
+# previously created by module gcp-oauth-long-access-strategy
+data "google_secret_manager_secret" "access_token" {
+  project   = var.project_id
+  secret_id = local.secret_id
+}
```

Comment:
I don't understand why you do this rather than re-use the existing one. This module doesn't do anything